### ISSUE TITLE
World creation end error

### ISSUE THREAD

**Issue by PavanLuca** [AFCMS/Subway-Miner#14]
```
Access denied. Reason: Kicked:
Initial world setup complete, please reconnect
```
at world creation end (a bit annoying, despite it only occur at world creation). Should be fixable as any other game that I run never throws an error once its world is created.

**Comment by AFCMS** [comment]
Fixed by this in upcoming (?) update https://github.com/AFCMS/Subway-Miner/pull/12/commits/d1d87c1c9ee464b037a6acc0a664fa8001005ffd.

Not sure when I will have time to finish and merge.

**Comment by AFCMS** [comment]
Fixed by 426fcda

Will publish the CDB update
